react-vite/base/package.json

```
@@ -34,8 +35,9 @@
     "eslint": "^8.35.0",
     "eslint-plugin-react": "^7.32.2",
     "jsdom": "^22.1.0",
+    "terser": "^5.4.0",
     "typescript": "^5.1.6",
     "vite": "^4.3.9",
     "vitest": "^0.32.2"
   }
-}
+}
```

```suggestion
}

```
Let's add the new line back.